### PR TITLE
Css update and unused image deletion

### PR DESCRIPTION
**Pull request by CoolCoder2019**
I have been cleaning up the CSS code, adding in a few things to make the website look cleaner and the code less of a hassle for others.

accept please :) 👍 🥇

### CODE REVIEW

File: htdocs/web_portal/css/web_portal.css [hunk 1]
```diff
@@ -40,15 +38,21 @@ background-color: #8dbdd8;
...
-body {
-	background-image: url('../img/background3.png');
-	background-repeat: no-repeat;
-	background-color: #d0dff9;
+body {  
+	background: linear-gradient(to bottom left,#F7F9FE, #DEE9FB, #D0DFF9);
 	color: #272A4B;
-	font-family: "VAGRoundedLtRegular",Tahoma, "museo-sans-1", "museo-sans-2", "Helvetica Neue", Geneva,sans-serif;
-	font-size: 1.3em;
+	font-family: 'PT Sans', sans-serif;
```
Comment (jrha) [inline htdocs/web_portal/css/web_portal.css]: To use this font you will need to add:
```html
<link href="https://fonts.googleapis.com/css?family=PT+Sans" rel="stylesheet">
```
to the `<head>` section of the main renderation code-a-majig.

Comment (GRyall) [reply htdocs/web_portal/css/web_portal.css]: Specifically edit htdocs\web_portal\static_php\standard_header.php, and add that line at the bottom of similar lines. You should then see the font change.

Comment (CoolCoder2019) [reply htdocs/web_portal/css/web_portal.css]: Now fixed

File: htdocs/web_portal/css/web_portal.css [hunk 2]
```diff
@@ -40,15 +38,21 @@ background-color: #8dbdd8;
 
 
 
-body {
-	background-image: url('../img/background3.png');
```
[the remark this answers is not in the document]
Comment (CoolCoder2019) [reply htdocs/web_portal/css/web_portal.css]: Sorted 👍

**Comment by jrha** [comment]
What about  `background.png` and `background2.jpg`, are they used?

File: htdocs/web_portal/static_php/standard_header.php [hunk 3]
```diff
@@ -25,9 +25,10 @@ function get_standard_header($title = null) {
         <link rel="stylesheet" href="'.GocContextPath::getPath().'javascript/datetimepicker/css/bootstrap-datetimepicker.min.css" />
         <link rel="stylesheet" href="'.GocContextPath::getPath().'javascript/bootstrap/css/bootstrap-select.min.css" />
         <link rel="stylesheet" type="text/css" href="'.GocContextPath::getPath().'css/web_portal.css" />
+	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT+Sans">
```
Comment (tofu-rocketry) [inline htdocs/web_portal/static_php/standard_header.php]: Should convert the tab to spaces to match the rest of the code.

**Comment by CoolCoder2019** [comment]
All unused images in the img directory have now been deleted.

**Pull request merged**